skopt/searchcv.py

```diff
@@ -272,7 +272,8 @@ def __init__(self, estimator, search_spaces=None, optimizer_kwargs=None,
                  n_iter=50, scoring=None, fit_params=None, n_jobs=1,
                  iid=True, refit=True, cv=None, verbose=0,
                  pre_dispatch='2*n_jobs', random_state=None,
-                 error_score='raise', return_train_score=True):
+                 error_score='raise', return_train_score=True,
+                 on_step=None):
```

Comment:
do we still need this now that we have callback support? I am not a fan of the name and also it is now not clear anymore if you should use this or the callbacks or both or ..?

Reply:
Oops nope we do not need it